engine/src/main/java/io/camunda/zeebe/engine/processing/timer/CronTimer.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.processing.timer;
+
+import io.camunda.zeebe.model.bpmn.util.time.Interval;
+import io.camunda.zeebe.model.bpmn.util.time.Timer;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Optional;
+import org.springframework.scheduling.support.CronExpression;
```

Comment:
❌ Please don't add an import of the spring library to the engine. Instead, please use the `cron-utils` library.